SConstruct

```diff
@@ -0,0 +1,6 @@
+# -*- python -*-
+from lsst.sconsUtils import scripts
+scripts.BasicSConstruct("spectractor",
+                        versionModuleName="python/version.py",
```

Comment:
How does spectractor return its version and should you write this file into `python/spectractor/version.py` so that you could get the eups version for this package?

Reply:
Currently it hard codes something into `_version.py` (note the leading underscore). I've moved this file, as you suggest, into that location, without the leading underscore, so they exist side-by-side for now, and I'll negotiate with upstream how to handle this in future.